### DISCUSSION TITLE
How to set sws_flags (full_chroma_int+bitexact+accurate_rnd) from pyAV

### DISCUSSION THREAD

**Discussion by jokla** [PyAV-Org/PyAV#1617] [1. Help] [answered]
Hi, 

I’m looking for some guidance on reading an MP4 video in YUV format and converting it to RGB without using hardware acceleration. I want to make sure I get consistent results across both ARM and x86 architectures. 

Could you please let me know how to set the `sws_flags`? I'm planning to use the following flags: `-sws_flags full_chroma_int+bitexact+accurate_rnd`. 

This is the full command line ([from here](https://stackoverflow.com/questions/70893502/why-does-ffmpeg-output-slightly-different-rgb-values-when-converting-to-gbrp-and)):
`ffmpeg -i video.mp4 -frames 1 -color_range pc -f rawvideo -pix_fmt rgb24 -sws_flags full_chroma_int+bitexact+accurate_rnd output_good.rgb24`

Thanks so much for your help! I really appreciate it!

**Comment by oakaigh** [comment, marked as answer]
See https://github.com/PyAV-Org/PyAV/issues/2307
